### DISCUSSION TITLE
Error when trying to create a new account

### DISCUSSION THREAD

**Discussion by Rcisidoro** [chatwoot/chatwoot#9794] [Self hosted]
I'm trying to create a new account for testing, when confirming, the following message returns in the log: NoMethodError (undefined method `feature_send_agent_name_in_whatsapp_message=' for an instance of Account):

I'm using V3.11.0, hosted on my infrastructure

**Comment by sojan-official** [comment]
Are you using official Chatwoot images or one with any customisations ?

**Reply by Rcisidoro** [reply]
I'm using the official image v3.11.0, my idea is to create an official company and another to test some of our company's marketing channels.

**Reply by thales-sz** [reply]
any update? Same error here
